erpnext/selling/report/lost_quotations/lost_quotations.json

```
@@ -0,0 +1,30 @@
+{
+ "add_total_row": 0,
+ "columns": [],
+ "creation": "2023-11-23 18:00:19.141922",
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "filters": [],
+ "idx": 0,
+ "is_standard": "Yes",
+ "letter_head": "ALYF GmbH",
```

```suggestion
 "letter_head": null,
```

why does this happen?